stackslib/src/chainstate/stacks/boot/pox_2_tests.rs

```diff
@@ -282,14 +282,14 @@ pub fn check_pox_print_event(
                 Some(v) => {
                     if v != &inner_val {
                         wrong.push((
-                            format!("{}", &inner_key),
-                            format!("{}", v),
-                            format!("{}", &inner_val),
+                            (&inner_key).to_string(),
```

Comment:
I think we can drop the reference and parens on this and others:
```suggestion
                            inner_key.to_string(),
```

Reply:
Fixed in 00fc87558